catalogs/sources/gtfs/schedule/us-california-alameda-contra-costa-transit-district-ac-transit-gtfs-1244.json

```diff
@@ -15,7 +15,7 @@
         }
     },
     "urls": {
-        "direct_download": "https://transitfeeds.com/p/ac-transit/1269/latest/download",
+        "direct_download": "https://api.actransit.org/transit/gtfs/download?token={{ API_KEY }}",
```

Comment:
As highlighted in the release plan, right now API-based sources can’t be added since we’re running checks to see if a source is a valid ZIP file. I’ve created [this issue](https://github.com/MobilityData/mobility-database-catalogs/issues/130) to make the checks conditional so API sources can be added and we can define a convention for including API keys and authorization methods in the schema.

I’ve also added this info to CONTRIBUTING.MD so the constraint is clearer.

Reply:
Addressed in updated PR.